src/plugins/log/log.js

```diff
@@ -81,9 +113,30 @@ Log.getInstance = function() {
   return this._instance
 }
 
+/**
+ * Specify a `level` to log.
+ * @method setLevel
+ * @param {Number} level valid level log [0,1,2,3,4].
+ */
 Log.setLevel = function(level) { this.getInstance().level = level }
 
+/**
+ * Set Log for Debug.
+ * @method debug
+ */
 Log.debug = function() { this.getInstance().debug.apply(this.getInstance(), arguments) }
+/**
+ * Set Log for Info.
+ * @method info
+ */
 Log.info = function() { this.getInstance().info.apply(this.getInstance(), arguments) }
+/**
+ * Set Log for Warn.
+ * @method warn
+ */
 Log.warn = function() { this.getInstance().warn.apply(this.getInstance(), arguments) }
+/**
+ * Set Log for Error.
+ * @method error
+ */
```

Comment:
Those methods are only static helpers to log messages with a given log level, not "setting the log" to a given level.